Source/RestoredState.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  RestoredState.swift
+//  RxBluetoothKit
+//
+//  Created by Kacper Harasim on 21.05.2016.
+//  Copyright © 2016 Polidea. All rights reserved.
+//
+
+import Foundation
+import CoreBluetooth
+
+/**
+ Convenience class which helps reading state of restored BluetoothManager
+ */
+public struct RestoredState {
+
+    /**
+     Internal restored state dictionary
+    */
+    public let restoredStateData: [String:AnyObject]
+
+    let bluetoothManager: BluetoothManager
```

Comment:
1. Consider `unowned` for bluetooth manger
2. Make it public? Maybe user would like to check it out. (Notice that user has access to `restoredStateData` which you pass in `init`)